index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
- [ ]  In the project requirement, It was stated that we add a class to our header and use the class name to style our header.

for example: 👇
```
<h1 class="head">Hello Microverse!</h1>
```

Reply:
Thank you for the example, I will change inmediatly